crates/test/mock-consensus/src/keyring.rs

```diff
@@ -0,0 +1,24 @@
+pub struct Keys {
+    pub consensus_signing_key: ed25519_consensus::SigningKey,
+    pub consensus_verification_key: ed25519_consensus::VerificationKey,
+}
```

Comment:
i avoided the map (see [above](https://github.com/penumbra-zone/penumbra/pull/3902/files#r1509543713)) so for now i'm stuffing the signing/verification keys into a structure. we can revisit this when we want multiple validators etc.

Reply:
also of note: identity keys are stubs atm, but eventually the identity keys should be put here too.

Reply:
Per previous discussions about the scope of auto_app_state, I don't think we'll ever need to keep those keys — tests that want to have access to the identity keys can construct their own app state.

Reply:
e66628697 addresses this